### ISSUE TITLE
Add different return types to heatmap endpoint

### ISSUE THREAD

**Issue by EPajares** [goat-community/goat#2189]
Currenlty, we are only returning geojson or geobuf when hitting the heatmap endpoint.
It would be good if the user could also receive:

- Geopackage
- Shapefile (as zip and with truncated column names)
- CSV

I would like to bypass geopandas. As I assume loading the geojson into geodataframe and then export it might be quite slow. Maybe we can explore if there are other ways to treat the geojson directly and have a hight performance. Some research on suitable libraries might be necessary for this first.

**Comment by metemaddar** [comment]
For csv the fastest way is to use numpy methods like [tofile()](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.tofile.html#numpy.ndarray.tofile) or [savetxt()](https://numpy.org/doc/stable/reference/generated/numpy.savetxt.html#numpy.savetxt). The complex part is writing coordinates about how to write them to be usable.
The csv package of python is also a good choice. It is readable and should be optimized. We need to check. But if the numpy works, we won't need it.

**Comment by EPajares** [comment]
Yes I guess CSV is not a big challenge. It is mostly the geopackage and shapefile that can make challenges. From what I found through chatgpt we could either use Fiona + Shapely, which might not make much sense as Geopandas uses both to perform the same. Or we could try using GDAL and ogr. This will definitely be the fastest from my experience but it might be interesting comparing it with geopandas.

Do you mind trying it? You can consult chatgpt on the installation of gdal/ogr they have a python binding.

**Comment by metemaddar** [comment]
We need to:

- [ ] Put the geometry for csv and xlsx to the end of the row
- [ ] fix total_population and reached_population
- [ ] rename 22_3 to 22.3
